assignment 5/day 6/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const files = ["./files/mongodb.txt",
+  "./files/expressjs.txt",
+  "./files/angularjs.txt",
+  "./files/nodejs.txt",
+  "./files/fullStack.txt"
+];
+
+function readFile(file, index) {
+  fs.readFile(file, 'utf8', function(
+    err, data) {
+    if (err) console.log(err);
```

Comment:
the statement in if block should come in new line and if block must always be enclosed in `{}`. Even if it is a single statement.